### PR TITLE
chore: use infra.ci.jenkins.io instead of ci.jenkins.io

### PR DESCRIPTION
**Pull request by dduportal**
Why using infra.ci?

* Private instance means safer credentials as we interact with cloud (and generate cloud costs)
* Finer control of the job properties and traits
* Separation of concerns: ci.jenkins.io is meant for another persona than Jenkins infra team members

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,62 @@
 # packer-images
 
-A repository containing a number of packer build configurations for AWS & Azure.
+A repository containing a number of packer build configurations to generate Jenkins agent templates for all the Jenkins controllers
+of the jenkins-infra project, such as ci.jenkins.io, infra.ci.jenkins.io, etc.
 
-## Requirements
+## Contributing
```
Comment (timja) [inline README.md:6]: better to make a separate CONTRIBUTING.md file that will be shown automatically by github on PRs

Comment (dduportal) [reply README.md:6]: Do you mind if I do this as a subsequent PR? The 40 min wait time for a given PR is a bit much 😅

Comment (timja) [reply README.md:6]: belongs in a separate PR imo

Comment (dduportal) [reply README.md:6]: https://github.com/jenkins-infra/packer-images/pull/88

Comment (dduportal) [reply README.md:6]: Fair, I've added the change here as build was failing.

Comment (timja) [reply README.md:6]: i meant that these should be separate PRs as CONTRIBUTING should be discussed and agreed whilst fixing the build and getting the Azure agents working is separate.

i.e. merge your fixes here, but keep the ways of working stuff separate

**Review by MarkEWaite — APPROVED** [review]

**Pull request merged**